src/write.jl

```diff
@@ -137,18 +137,22 @@ function Base.write(
     p::Presentation;
     overwrite::Bool=false,
     open_ppt::Bool=true,
-    template_path::String=joinpath(TEMPLATE_DIR, "no-slides.pptx"),
+    template_path::Union{String, Vector{UInt8}}=DEFAULT_TEMPLATE_DATA,
 )
-
-    template_path = abspath(template_path)
-    template_isfile = isfile(template_path)
-
-    if !template_isfile
-        error(
-            "No file found at template path: $(repr(template_path))",
-        )
+    template_data = if template_path isa AbstractString
```

Comment:
With a `if x isa Y` pattern I would generally advise to just dispatch on the types with a function, that's more Julian. 

You know:
```julia
function read_template(template_path::AbstractString)
   # code
end

read_template(template_data::Vector{UInt8}) = template_data
```

(Also I didn't know that if-statements have a return variable)